packages/zwave-js/src/lib/controller/Inclusion.ts

```diff
@@ -171,6 +182,13 @@ export type InclusionOptionsInternal =
 		provisioning: PlannedProvisioningEntry;
 	};
 
+// BUGBUG: better way to do this?
+export type InclusionFlagsInternal = {
```

Comment:
This is the "preserver the flags used at START inclusion for STOP inclusion" struct. I'm new to typescript, I wasn't sure if there was a better way or something. Does this need to be a type? Or just an Interface?